src/components/Charts/ModelChart.style.js

```diff
@@ -224,6 +224,19 @@ export const Wrapper = styled.div`
           ? COLOR_MAP.GREEN.DARK
           : props.projections.getAlarmLevelColor()};
     }
+
+    &.custom-plot-label-${snakeCase(
+        INTERVENTIONS.SHELTER_IN_PLACE_WORST_CASE,
+      )} {
+      background: ${props =>
+        props.projections.getAlarmLevelColor() === COLOR_MAP.GREEN.BASE
+          ? COLOR_MAP.GREEN.DARK
+          : props.projections.getAlarmLevelColor()};
+    }
+
+    &.custom-plot-label-${snakeCase(INTERVENTIONS.SHELTER_IN_PLACE)} {
+      background: ${props => props.projections.getAlarmLevelColor()};
+    }
```

Comment:
I'm guessing this is the iffy css merge you mentioned?

Reply:
Yep, fixed.